webapp/channels/src/components/integrations/abstract_outgoing_webhook.tsx

```diff
@@ -568,7 +589,7 @@ export default class AbstractOutgoingWebhook extends React.PureComponent {
                                 className='btn btn-primary'
                                 type='submit'
                                 spinning={this.state.saving}
-                                spinningText={localizeMessage(this.props.loading.id, this.props.loading.defaultMessage)}
+                                spinningText={localizeMessage(this.props.loading.id as string, this.props.loading.defaultMessage as string)}
```

Comment:
**Note: ** I ended up following what was done over in `abstract_incoming_webhook.tsx`: https://github.com/mattermost/mattermost/blob/96d3e5864f95065e4845101ad3cabc9cb9a58f35/webapp/channels/src/components/integrations/abstract_incoming_webhook.tsx#L382C4-L382C4

It seems like we should always have `loading.id` and `loading.defaultMessage` given the usage, but the typing on `MessageDescriptor` has an optional `id`.